src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -10,15 +10,15 @@
 {{#all_user_clusters}}
   {{~side}} cluster {{asUpperCamelCase name}} = {{code}} {
   {{#zcl_enums}}
-  enum {{asUpperCamelCase name}} : {{type}} {
+  enum {{asUpperCamelCase name}} : ENUM{{multiply size 8}} {
```

Comment:
The enum and bitmap tables no longer have a column for type. All enums and bitmaps including atomic enums and bitmaps now exist in the enum and bitmap tables. All the enums and bitmaps have a size column which can be used to determine any kind of enum/bitmap type.